### PR TITLE
feat: add session_id tag to sentry

### PR DESCRIPTION
**Pull request by ErickSharp**
## Summary of Changes
Adds a new Sentry tag named 'session_id' that is generated using nanoid so that users can show it to support when they encounter a bug. On flypadosv3, this session ID will appear when the flypad itself crashes, and also will be available in the About settings page.

## Additional context

Discord username (if different from GitHub): Erick [Z-6]#6484

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/index.tsx
Comment (beheh) [reply src/instruments/src/EFB/index.tsx:87]: @Benjozork Are you suggesting we should hold off on proceeding with this PR until that has been addressed, or can this land regardless?

**Comment by beheh** [comment]
I've added a tag to hold off on merging for the time being.

@ErickSharp Any objection to moving this back to draft until the feedback from Benjozork above has been addressed?

**Comment by ErickSharp** [comment]
> I've added a tag to hold off on merging for the time being.
> 
> @ErickSharp Any objection to moving this PR back to draft until the feedback from Benjozork above has been addressed?

Yes sorry I forgot about this PR. I'll make the changes necessary soon :)

**Review by frankkopp — APPROVED** [review]

File: src/instruments/src/EFB/index.tsx
Comment (beheh) [inline src/instruments/src/EFB/index.tsx:56]: Won't this always clear the `A32NX_SENTRY_SESSION_ID` persistent property when the EFB intiializes?

As the `generatedSessionID` below will only be set once on a JS load, before the EFB renders.

Comment (ErickSharp) [reply src/instruments/src/EFB/index.tsx:56]: > Won't this always clear the `A32NX_SENTRY_SESSION_ID` persistent property when the EFB intiializes?
> 
> 
> 
> As the `generatedSessionID` below will only be set once on a JS load, before the EFB renders.

Wouldn't the cleanup function for the useffect run only when the component gets unmounted?

Comment (beheh) [reply src/instruments/src/EFB/index.tsx:56]: Ah, you're right, I missed the double-arrow.

**Pull request merged**
